Source/WTF/wtf/PlatformEnable.h

```diff
@@ -890,7 +890,8 @@
 #endif
 #endif
 
-#if !defined(ENABLE_PDFJS) && (PLATFORM(COCOA) || PLATFORM(GTK) || PLATFORM(WPE))
+// FIXME: Reenable PDFJS by default for Cocoa (https://bugs.webkit.org/show_bug.cgi?id=235981).
```

Comment:
I just filed https://bugs.webkit.org/show_bug.cgi?id=242263 for this, can we reference this instead?